### PR TITLE
Add bin links

### LINKED ISSUE #56: Support node_modules/.bin when linking deps
It would be nice if linked packages provide correct dependencies in `node_modules/.bin`. This helps when manually running and debugging scenarios.

### PR DESCRIPTION
**Pull request by krisselden**
Fix #56

### CODE REVIEW

**Comment by krisselden** [comment]
I still need to add test but it is not running tests on my PR

File: index.ts [hunk 1]
```diff
@@ -270,6 +271,15 @@ export class Project {
     }
   }
 
+  async binLinks() {
+    await binLinks({ pkg: this.pkg, path: this.baseDir, top: false, global: false });
+  }
+
+  async write() {
```
Comment (thoov) [inline index.ts:278]: This looks good. We should probably deprecate calling `project.writeSync()` in favor of this. We will need to update the README with new examples of `await project.write()`.

Comment (krisselden) [reply index.ts:278]: in a library like this what the best way to deprecate? console.warn? just documentation with jsdoc @deprecate?

**Comment by krisselden** [comment]
Added a test.

File: index.ts [hunk 2]
```diff
@@ -253,23 +254,45 @@ export class Project {
...
+    this.writeProject();
+    await this.binLinks();
+  }
+
+  /**
+   * @deprecated please use `await project.write()` instead.
+   */
   writeSync() {
+    console.warn('this method is deprecated, please use write instead');
```
Comment (thoov) [inline index.ts:266]: We should probably use your deprecated message from above which includes the exact method people should use:

```
project.writeSync() is now deprecated, please use `await project.write()` instead.
```

**Comment by thoov** [comment]
LGTM!

**Pull request merged**